src/test/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisResultServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package org.gridsuite.securityanalysis.server.service;
+
+import com.vladmihalcea.sql.SQLStatementCountValidator;
+import org.gridsuite.securityanalysis.server.dto.SecurityAnalysisStatus;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.UUID;
+
+import static com.vladmihalcea.sql.SQLStatementCountValidator.assertDeleteCount;
+import static org.gridsuite.securityanalysis.server.SecurityAnalysisProviderMock.RESULT;
+
+/**
+ * @author Florent MILLOT <florent.millot@rte-france.com>
+ */
+@SpringBootTest
+class SecurityAnalysisResultServiceTest {
+
+    @Autowired
+    SecurityAnalysisResultService securityAnalysisResultService;
+
+    @Test
+    void deleteResultPerfTest() {
+        UUID resultUuid = UUID.randomUUID();
+        securityAnalysisResultService.insert(resultUuid, RESULT, SecurityAnalysisStatus.CONVERGED);
+        SQLStatementCountValidator.reset();
+
+        securityAnalysisResultService.delete(resultUuid);
+
+        // 5 manual deletes + the list of ContingencyElementEmbeddable inside ContingencyEntity
```

Comment:
Then you see here that there will be a supplementary delete request for each ContingencyEntity

Reply:
actually it's done with a smart sub select but, we already have the list of those Uuids, then maybe we should remove in this table manually as well ?
```
delete from contingency_entity_contingency_elements where contingency_entity_uuid in(select c1_0.uuid from contingency c1_0 where c1_0.result_id=$1)
```